app/urls.py

```diff
@@ -0,0 +1,7 @@
+from django.contrib import admin
+from django.urls import path, include
+
+urlpatterns = [
+    path("admin/", admin.site.urls),
+    path("api/cinema/", include("cinema.urls", namespace="cinema")),
```

Comment:
The `include` function is used with a `namespace` argument, but the `cinema.urls` module must define an `app_name` attribute for namespacing to work correctly. Ensure that `cinema/urls.py` has `app_name = 'cinema'` defined at the top of the file.